template/en/default/request/queue.json.tmpl

```diff
@@ -0,0 +1,42 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0. #%]
+
+[% RAWPERL %]
+my @display_columns = (
+    "requester", "requestee",      "type",    "status",  "bug_id", "bug_summary",
+    "attach_id", "attach_summary", "ispatch", "created", "category"
+);
+my $requests    = $stash->get('requests');
+my $time_filter = $context->filter('time', [ '%Y-%m-%dT%H:%M:%SZ', 'UTC' ]);
```

Comment:
Why UTC and not the user's timezone?

Reply:
I just copy-pasted this from somewhere else but it actually doesn't matter because the frontend JavaScript code parses the string with `Date()` that can retrieve the local time easily. I think, for API response payloads, it's safe to always use UTC.